src/Game/ResearchAPI.c

```diff
@@ -998,12 +989,12 @@ void rmAPIStartup(void)
     scriptSet(NULL,"ResearchDepend.script",TechnologyDependancies);
 #endif
 
-    rmSetPrintList(RM_SPR1,&SinglePlayerR1TechStatic);
-    rmSetPrintList(RM_SPR2,&SinglePlayerR2TechStatic);
-    rmSetPrintList(RM_MPR1,&MultiPlayerR1TechStatic);
-    rmSetPrintList(RM_MPR2,&MultiPlayerR2TechStatic);
-    rmSetPrintList(RM_MPCR1,&MultiPlayerCR1TechStatic);
-    rmSetPrintList(RM_MPCR2,&MultiPlayerCR2TechStatic);
+    scriptSetPrintList(NULL,"ResearchList_SPR1.script",SingleplayerR1TechListBuilder);
+    scriptSetPrintList(NULL,"ResearchList_MPR1.script",MultiplayerR1TechListBuilder);
+    scriptSetPrintList(NULL,"ResearchList_MPCR1.script",MultiplayerCR1TechListBuilder);
+    scriptSetPrintList(NULL,"ResearchList_SPR2.script",SingleplayerR2TechListBuilder);
+    scriptSetPrintList(NULL,"ResearchList_MPR2.script",MultiplayerR2TechListBuilder);
+    scriptSetPrintList(NULL,"ResearchList_MPCR2.script",MultiplayerCR2TechListBuilder);
 
     rmGUIStartup();
```

Comment:
Something tells me that all of these `scriptSetPrintList` calls should go to the `rmGUIStartup` block.